Common/MathUtils/include/MathUtils/RandomRing.h

```diff
@@ -21,13 +21,12 @@
 /// The numbers can then be used as a continuous stream in
 /// a ring buffer
 ///
-/// origin: TPC
 /// @author Jens Wiechula, Jens.Wiechula@cern.ch
 
-#ifndef ALICEO2_TPC_RANDOMRING_H_
-#define ALICEO2_TPC_RANDOMRING_H_
+#ifndef ALICEO2_MATHUTILS_RANDOMRING_H_
+#define ALICEO2_MATHUTILS_RANDOMRING_H_
 
-#include <boost/format.hpp>
+// #include <boost/format.hpp>
```

Comment:
if it is not used why not remove it completely?

Reply:
agreed